src/gpodder/plugins/soundcloud.py

```diff
@@ -38,12 +38,39 @@
 
 
 # gPodder's consumer key for the Soundcloud API
-CONSUMER_KEY = 'zrweghtEtnZLpXf3mlm8mQ'
+CLIENT_ID = "foo"
+CLIENT_SECRET = "bar"
 
+access_token = None
+token_expire = datetime.now()
 
 logger = logging.getLogger(__name__)
 
 
+class SoundcloudException(Exception):
+    pass
+
+
+def urlopen_with_token(url):
```

Comment:
In theory, this method could have more parameters, but as all occurrences only use 1 parameter, I didn't bother to implement more, unused parameters, that are passed to urlopen then